### DISCUSSION TITLE
One-to-any interface

### DISCUSSION THREAD

**Discussion by ghost** [directus/directus#4525] [Feature Requests]
I'm currently doing this by manipulating multiple one-to-many interfaces, but support in v-form for such alias interface would be nice.

**Comment by rijkvanzanten** [comment]
@xl-1200 Could you clarify what a one-to-any would look like? We've been discussing all the various different names and directions, but I just want to make sure that "your" one-to-any is the same as what we'd consider a one-to-any 😁

**Reply by ghost** [reply]
My current one-to-any is a hack where I just manipulate multiple one-to-many interfaces (in vue by watching and setting `$parent.$parent.$parent.values` in the form). Visually it is a table (same as the one-to-many, but that may change with #3647), but it also shows collection icon/name on the left.

I think ideally it would be 1 alias interface with its form values of the same format as an M2M or M2A - that is, a CSV of IDs - except those aren't IDs to junction collections, they're IDs of the children collections (the ones with the FKs). And the same create/update logic would apply (rather than primitive ID, value in CSV becomes an object, etc.). If this isn't possible with the current API I don't know if having it be a special case is worth it to be frank.
